### PR TITLE
Delete custom op for adamw.

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

Function optimization
### PR changes

APIs
### Description

Now, Paddle support adamw as op, so we can delete the custom AdamW.
https://github.com/PaddlePaddle/Paddle/pull/35020

### CODE REVIEW

File: examples/language_model/gpt-3/static/run_pretrain_static.py [hunk 1]
```diff
@@ -294,33 +294,17 @@ def do_train(args):
...
+            optimizer = paddle.optimizer.AdamW(
+                learning_rate=lr_scheduler,
+                beta1=args.adam_beta1,
+                beta2=args.adam_beta2,
+                epsilon=args.adam_epsilon,
+                grad_clip=clip,
+                weight_decay=args.weight_decay,
+                apply_decay_param_fun=lambda x: x in decay_param)
+            # alias
+            optimizer.apply_optimize = optimizer._apply_optimize
```
Comment (wawltor) [inline examples/language_model/gpt-3/static/run_pretrain_static.py:307]: 这里是个不兼容的修改，在文档中注明一下使用的paddlepaddle的版本吧

File: examples/language_model/gpt/run_pretrain_static.py [hunk 2]
```diff
@@ -294,33 +294,18 @@ def do_train(args):
...
+                learning_rate=lr_scheduler,
+                beta1=args.adam_beta1,
+                beta2=args.adam_beta2,
+                epsilon=args.adam_epsilon,
+                grad_clip=clip,
+                weight_decay=args.weight_decay,
+                apply_decay_param_fun=lambda x: x in decay_param)
+
+            # alias
+            optimizer.apply_optimize = optimizer._apply_optimize
```
Comment (wawltor) [inline examples/language_model/gpt/run_pretrain_static.py:308]: 同上

File: examples/language_model/gpt-3/static/run_pretrain_static.py (hunk 1 above)
Comment (ZHUI) [reply examples/language_model/gpt-3/static/run_pretrain_static.py:307]: Done

File: examples/language_model/gpt/run_pretrain_static.py (hunk 2 above)
Comment (ZHUI) [reply examples/language_model/gpt/run_pretrain_static.py:308]: Done

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**